streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfo.java

```diff
@@ -115,6 +119,10 @@ private SubscriptionInfo(final SubscriptionInfoData subscriptionInfoData) {
         this.data = subscriptionInfoData;
     }
 
+    public int errorCode() {
+        return data.errorCode();
```

Comment:
What happens if we try to read the error code of an earlier subscription version? I genuinely don't know what the generated code does, but we should make sure it doesn't throw an NPE or something. Could you add a unit test for this case?

Reply:
Good idea. It does not seem to do anything. but good to have a test for it